### ISSUE TITLE
[BUG] Bulk delete of multiple date-groups of photos fails

### ISSUE THREAD

**Issue by ethanxrosen** [immich-app/immich#2263]
### The bug

When selecting and deleting a single pic or single group of pics from the main Photos tab delete will work as expected. However, if multiple groups of photos are selected (for example both "Sun, Apr 16, 2023" and "Sat, Apr 15, 2023") delete doesn't seem to work.

For me on web nothing happens after delete is pressed; on iOS the groups temporarily disappear but reappear after a refresh.

### The OS that Immich Server is running on

Ubuntu 22.10

### Version of Immich Server

Immich 1.53.0

### Version of Immich Mobile App

1.53.0 build 93

### Platform with the issue

- [X] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
hosted via cloudron
https://github.com/cloudron-io/immich/blob/main/docker/docker-compose.yml
```

### Your .env content

```Shell
https://github.com/cloudron-io/immich/blob/main/docker/.env.example
```

### Reproduction steps

```bash
1. Select many (>=2) "date groups" of pics from the Photos tab on either iOS app or web
2. Attempt to delete & hit confirm
3. Refresh
4. Photos still remain
```

**Comment by alextran1502** [comment]
This is a known bug if the assets are in album it cannot be deleted from the timeline. It is already fixed and will be available in the next release

**Comment by alextran1502** [comment]
fixed in #2223

**Comment by ethanxrosen** [comment]
amazing- thank you Alex
